cli/cmd/encore/daemon/migrations/1_initial_schema.up.sql

```diff
@@ -1,10 +1,3 @@
-CREATE TABLE IF NOT EXISTS schema_migrations (
-    dummy BOOLEAN PRIMARY KEY,
-    version INT NOT NULL
-);
-
-INSERT OR REPLACE INTO schema_migrations(dummy, version) VALUES (true, 1);
-
```

Comment:
Should we issue a drop command for this older db?

Reply:
We do issue a `DROP TABLE schema_migrations` but it's being done in code (since it needs to run before we run the migrations)